S21/myadlapalli/Language.py

```diff
@@ -0,0 +1,72 @@
+NO_CHANGE_LANGS = ['zh', 'ja', 'th']
+TURKISH_LANGS = ['tr', 'az']
+
+IRISH = 'ga'
+IRISH_VOWELS = ['A', 'E', 'I', 'O', 'U', 'Á', 'É', 'Í', 'Ó', 'Ú']
+
+GREEK = 'el'
+
+
+def toLower(sentence, lang):
+    sentence = sentence.strip()
+    result = ''
+    lang = lang.split('-')[0]
+
+    # Languages with no change
+    if lang in NO_CHANGE_LANGS:
+        return sentence
+
+    words = sentence.split(" ")
+    # print(words)
+
+    for word in words:
+        result += ' ' + wordToLower(word, lang)
+
+    return result.strip()
+
+def wordToLower(word, lang):
+    result = ''
+    # Languages with no change
+    if lang in NO_CHANGE_LANGS:
```

Comment:
We have already performed this condition, here it can just increase the complexity of the code.